packages/garbo/src/familiar/barfFamiliar.ts

```diff
@@ -77,10 +77,6 @@ const SPECIAL_FAMILIARS_FOR_CACHING = new Map<
   ],
   // Uniquely required to equip its fam equip to meaningfully have value
   [$familiar`Jill-of-All-Trades`, { equip: $item`LED candle` }],
-  [
-    $familiar`Skeleton of Crimbo Past`,
-    { equip: $item`small peppermint-flavored sugar walking crook` },
```

Comment:
I keep flip flopping as to whether this is necessary. I guess dumb answer is try it both ways?